Makefile

```diff
@@ -112,6 +114,80 @@ enwiki_tuning_reports: \
 	tuning_reports/enwiki.wp10.md \
 	tuning_reports/enwiki.nettrom_wp10.md
 
+########################## Persian Wikipedia ###################################
+
+# https://quarry.wmflabs.org/query/25457
+datasets/fawiki.sampled_revisions.300.json:
+	wget https://quarry.wmflabs.org/run/244123/output/0/json-lines?download=true -qO- > $@
+
+datasets/fawiki.labelings.300.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/fawiki/71/ > $@
+
+datasets/fawiki.labeled_revisions.600.json: \
+		datasets/fawiki.sampled_revisions.300.json \
+		datasets/fawiki.labelings.300.json
+	cat $^ > $@
+
+datasets/fawiki.labeling_revisions.w_text.600.json: \
+		datasets/fawiki.labeled_revisions.600.json
+	cat $< | \
+	./utility fetch_text \
+	  --api-host=https://fa.wikipedia.org --threads 4 \
+	  --verbose > $@
+
+datasets/fawiki.labeling_revisions.w_cache.600.json: \
+		datasets/fawiki.labeling_revisions.w_text.600.json
+	cat $< | \
+	./utility extract_from_text \
+	  wikiclass.feature_lists.fawiki.wp10 \
+	  --verbose > $@
+
+
+tuning_reports/fawiki.wp10.md: \
+		datasets/fawiki.labeling_revisions.w_cache.600.json
+	cat $< | \
+	revscoring tune \
+	  config/classifiers.params.yaml \
+	  wikiclass.feature_lists.fawiki.wp10 \
+	  wp10 \
+	  accuracy.macro \
+	  --pop-rate '"e"=0.7314705724717468' \
+	  --pop-rate '"bd"=0.2314879676843963' \
+	  --pop-rate '"b"=0.03023005873940185' \
+	  --pop-rate '"a"=0.0029374402333403227' \
+	  --pop-rate '"ba"=0.002439090897978488' \
+	  --pop-rate '"adq"=0.00143486997313615' \
```

Comment:
Looks like this is using frwiki classes.  Must be a WIP

Reply:
Yes, On it